doc/glossary.rst

```diff
@@ -14,6 +14,16 @@
                 system interface. Since Luminous (12.2), BlueStore has been
                 Ceph's default and recommended storage back end.
 
+        Bucket
```

Comment:
bucket is a term from aws S3, so it's probably helpful to refer to some of their documentation. for example,

https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-buckets-s3.html
https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingBucket.html

i don't think it's useful to say anything about zones or multisite in the basic definition of a bucket

Reply:
I'll remove all of this what-contains-what-relative-to-each-other stuff when I have "object", "bucket", "zone", "zonegroup", and "realm" defined in the glossary. Think of this as lexicographical scaffolding.

Reply:
I'll add the AWS references after a nap.

Reply:
I think about that sometimes when writing about RGW, but my sense is that so few people are using Swift that it wasn't worth the complexity. ymmv.

Reply:
https://github.com/ceph/ceph/pull/50246 - this PR adds AWS and OpenStack information about what RGW calls "buckets" (and this PR adds the information to the definition of bucket that Casey wanted me to add) (and it also removes the zone-and-zonegroup-related where-in-the-hierarchy-do-buckets-fall information that Casey deemed to be irrelevant and which, after a night's sleep and greater familiarity with the family of concepts here discussed, I have come around on)